bin/run

```diff
@@ -28,4 +28,9 @@ if ! "${conda}" env list | grep -q gedi_subset; then
 fi
 
 [[ -n "${quiet}" ]] || set -x
-"${conda}" run --live-stream --name gedi_subset "${conda_run_args[@]}"
+
+if [[ "${CI:-}" == "true" ]]; then
+    "${conda_run_args[@]}"
+else
+    "${conda}" run --live-stream --name gedi_subset "${conda_run_args[@]}"
```

Comment:
looks like `--live-stream` is being deprecated https://github.com/conda/conda/issues/11209
change to `--no-capture-output` ? not sure if that has the same intent.

Reply:
Too funny. A while back, I was using `--no-capture-output`. I don't recall why I landed on using `--live-stream` instead, but I'll switch back :-)